dev_tools/scripts/create-s3-buckets.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+if [[ ! -f s3cmd.conf ]]; then
```

Comment:
If these scripts are intended to be run locally, since they are in the `dev_tools` folder you might want to keep them posix compliant. Replace `[[` with  `[`.

Reply:
Ok